docs/functions/tm_flatten.md

```diff
@@ -25,80 +25,6 @@ tm_flatten([[["a", "b"], []], ["c"]])
 
 Indirectly-nested lists, such as those in maps, are _not_ flattened.
 
-## Flattening nested structures for `for_each`
```

Comment:
those examples uses the Terraform `for_each` and do not make sense for Terramate.